### PR TITLE
Attach and create ROIs

### PR DESCRIPTION
**Pull request by manics**
- Uses in-place upload to attach mask tiffs
- Creates mask ROIs

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,6 @@ flake8
 yamllint
 ome-model==6.0.0
 Pillow
+scikit-image
+https://github.com/manics/omero-upload/archive/pr1.zip
```
Comment (sbesson) [inline requirements.txt]: This should be able to consume `omero-upload==0.2.0`

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,6 @@ flake8
...
+https://github.com/manics/omero-rois/archive/master.zip
```
Comment (sbesson) [inline requirements.txt]: I assume we will want https://github.com/ome/omero-rois deployed on PyPI first. Given https://github.com/ome/omero-rois/releases, do we want to go for `0.3.0` or `0.2.1` ?

Comment (manics) [reply requirements.txt]: `0.3.0` since master is on `0.3.0.dev0`. Has the naming discussion been resolved?

Comment (sbesson) [reply requirements.txt]: :+1: for the version number
Re naming if you are talking about `omero-rois` vs `omero-roi`, no-one commented on my suggestion so I do not think there is a resolution. We can certainly go ahead and get `omero-rois` published and rename later, the main downside will be the PyPI project might change and unlike GitHub, PyPI does not seamlessly redirect.

**Review by sbesson — APPROVED** [review]
Confocal images were re-annotated using the latest version of this script

Screen Shot 2019-04-03 at 13 47 53

With  64cda7b8f045d79c50e3bb523ffbcac6e53edae9, these scripts are now consuming changes moved upstream.

**Pull request merged**